### ISSUE TITLE
Supporting extra worlds with corresponding gamemode addons

### ISSUE THREAD

**Issue by wellnesscookie** [BentoBoxWorld/InvSwitcher#5]
**Is your feature request related to a problem? Please describe.**
Yes, it is probably is. Did not test, but the addon probably changes your inventory and food stats if you enter a non BBox Gamemode world. 

**Describe the solution you'd like**
I'd like if it was possible to implement a feature that you can create a world starting with bskyblock_[name] and that the InventoryAddon treats it like an extra world of bskyblock (It is being recognised and sorted due to **bskyblock** prefix). The point was to auto-connect the inventory, xp and food of BSkyBlock worlds with the new-created one as well.

**For an example:**

Now you would have your inventory shared between:
- bskyblock
- bskyblock_the_nether
- bskyblock_the_end
- bskyblock_[new_world]

Tho it doesn't has to be done this way, but I tried to think of an easy solution for this to be supported too. By the way... Of course it doesn't has to be "bskyblock", it can just take the name of the worlds from gamemodes config.yml. 

Example of a world name required to exist to so it would be recongised and connectd with the corresponding group of worlds: 
`[gamemode_addon_name]_[any-world-name]'

**Additional context:**
If it can't be done this way, hope some1 will find another way to do it.

**Comment by tastybento** [comment]
I think what you are asking for is grouping of worlds? So the inventory would stay the same in all those worlds?

**Comment by wellnesscookie** [comment]
@tastybento 

Yes, exactly. The reason I wrote that much was because I wanted to maybe present a potential easy way of grouping them inside a box of gamemode corresponding worlds, but anyway... I'm not yet good or close to being good at coding, nor forseeing what is possible and what is not, so please don't mind me.

**Comment by wellnesscookie** [comment]
@tastybento 

This was what I thought:
https://github.com/wellnesscookie/addon-invSwitcher/commit/fe7df496430e3bbd374116b6629235bac7635325

I generated a world named after skyblock_endparty and added those 2 string replace methods and tested - it worked. It's just an extra world grouped with skyblock worlds, tho this is hard-coded, it would need not to be.

**Comment by tastybento** [comment]
Yep, that would be the way to do it. I'll look into how to make it generic.

**Comment by Magnum97** [comment]
I also hoped this would be possible.

I see this was added to the 0.0.5 milestone. Current version is 1.11.0 - does that mean it is a feature now?
